### ISSUE TITLE
demo run error

### ISSUE THREAD

**Issue by liushiqi1001** [halturin/ergo#51]
```
$ pwd
/Users/shiqi/go/src/github.com/ergo/examples/genserver

$ go run demoGenServer.go
Init: args [] 
Run erl shell:
erl -name erl-demo@127.0.0.1 -setcookie 123
-----Examples that can be tried from 'erl'-shell
gen_server:cast({example,'demo@127.0.0.1'}, stop).
gen_server:call({example,'demo@127.0.0.1'}, hello).
```

when i run erl shell 
```
$ erl -name erl-demo@127.0.0.1 -setcookie 123
Erlang/OTP 23 [erts-11.1.3] [source] [64-bit] [smp:12:12] [ds:12:12:10] [async-threads:1] [hipe] [dtrace]

Eshell V11.1.3  (abort with ^G)
(erl-demo@127.0.0.1)1> gen_server:cast({example,'demo@127.0.0.1'}, stop).
ok
(erl-demo@127.0.0.1)2> =ERROR REPORT==== 21-May-2021::17:45:03.682489 ===
** 'erl-demo@127.0.0.1': Connection attempt to node 'demo@127.0.0.1' aborted since it cannot handle ["BIG_CREATION"].**

gen_server:call({example,'demo@127.0.0.1'}, hello).
=ERROR REPORT==== 21-May-2021::17:45:12.825623 ===
** 'erl-demo@127.0.0.1': Connection attempt to node 'demo@127.0.0.1' aborted since it cannot handle ["BIG_CREATION"].**

** exception exit: {{nodedown,'demo@127.0.0.1'},
                    {gen_server,call,[{example,'demo@127.0.0.1'},hello]}}
     in function  gen_server:call/2 (gen_server.erl, line 238)
(erl-demo@127.0.0.1)3> 

```

go version 1.15.1

**Comment by halturin** [comment]
Hello, thanks for your report.

It doesn't support Erlang 23 and above and it was mentioned in README.md:

> Support Erlang 22. (including fragmentation feature)

**Comment by halturin** [comment]
I closed this issue because of #49
